hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -1021,6 +1021,49 @@ function upgrade_1023() {
     return true;
   }
 
+  /**
+   * Creates Option Group and Option Values for Health an Life Insurance Plan 
+   * Types.
+   * 
+   * @return boolean TRUE
+   */
+  public function upgrade_1025() {
+    $this->installInsurancePlanTypes();
```

Comment:
maype this could be renamed to createInsurePlanTypes  since install keyword is rarely used for method names that create sutff in civihr